app/src/main/java/org/simple/clinic/contactpatient/ContactPatientProfile.kt

```diff
@@ -59,4 +59,13 @@ data class ContactPatientProfile(
         entityColumn = "patientUuid"
     )
     val bloodPressureMeasurement: BloodPressureMeasurement?
-) : Parcelable
+) : Parcelable {
+
+  fun withoutDeletedBusinessIds(): ContactPatientProfile {
+    return copy(businessIds = businessIds.filter { it.deletedAt == null })
+  }
```

Comment:
We can remove this if we are not using business IDs in contact patient sheet.

Reply:
We're using this in the effectHandler, ptal?

Reply:
https://github.com/simpledotorg/simple-android/pull/2888#discussion_r678211732